### DISCUSSION TITLE
500张图片训练600epoch，ppocr检测模型v3预训练，指标一直很低，并且检测框是图片的边缘框，不是字的框

### DISCUSSION THREAD

**Discussion by jingmingtao** [PaddlePaddle/PaddleOCR#12602] [Q&A]
500张图片训练600epoch，ppocr检测模型v3预训练，指标一直很低，并且检测框是图片的边缘框，不是字的框，有大神知道为什么吗

**Comment by jingmingtao** [comment]
我的数据可能有些特别，一张图片只有两行，图片分辨率大约160x450(w,h)，字比较大几乎是文字每行宽度是75，一共两行。是这个原因吗？我尝试了许多的策略，没有解决。

**Comment by laishenghui** [comment]
你的问题解决了吗？
我也有类似的情况，我的一张图片只有一行
